bitsv/network/services/bitindex3.py

```diff
@@ -30,11 +30,14 @@ def _get_authorized_headers(self):
         return headers
 
     def get_utxos(self, address, sort=None):
-        """
-        Retrieve utxos for address
+        """Gets all unspent transaction outputs belonging to an address.
 
+        :param address: The address in question.
+        :type address: ``str``
+        :raises ConnectionError: If all API services fail.
         :param address: Address to get utxos for
-        :param sort: Format is 'field:asc' such as 'value:desc' to sort by value descending
+        :param sort: 'value:desc' or 'value:asc' to sort unspents by descending/ascending order respectively
```

Comment:
Why/where would this be used? There's all kinds of possible sorts you might want on unspents and I think we already have code that does this. Maybe that's been changed, but there should be some kind of unspent selection logic that would have this.

Reply:
This is just here because it is a parameter that the BitIndex3 API supports.

Reply:
I see. So what do we do if passing sort to a different API?

Reply:
I am honestly not sure. Other APIs might be vastly different, including different methods and responses.

I guess here is where we need to make some design decisions:
1. Do we want our API integrations to be one-to-one implementations or normalized?
   - If they are normalized, do we want to implement it in such a way as to provide the full feature set of each?
   - If they are not normalized at this layer, should we provide a normalization layer?
2. Do we want our integrations to primarily be consumed internally or to also be available as standalone services?

Reply:
Good point @teran-mckinney. How stupid of me... and @joshua-s agreed. See #24 for how I am thinking we go about this at present. (I think we keep NetworkAPI to bread and butter functions only and normalize the apis for this purpose but also provide the apis as standalone, unaltered services). Hopefully keeps everybody happy that way :)